BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -499,11 +499,21 @@ Cryptographic algorithms weaken over time as cryptanalysis improves: algorithms
                with the string "TLS_RSA_WITH_*", have several drawbacks, especially
                the fact that they do not support forward secrecy.
             
+* Implementations SHOULD NOT negotiate cipher suites based on
+               non-ephemeral finite-field Diffie-Hellman key agreement.
+
+  Rationale: These cipher suites, which have assigned values starting
+               with the string "TLS_DH_*", have several drawbacks, especially
+               the fact that they do not support forward secrecy.
 
 * Implementations MUST support and prefer to negotiate cipher suites 
                offering forward secrecy, such as those in the Ephemeral 
                Diffie-Hellman and Elliptic Curve Ephemeral Diffie-Hellman ("DHE" 
-               and "ECDHE") families.
+               and "ECDHE") families.  However, given the known limitations
+               around both construction (e.g., {{RACCOON}}) and negotiation
+               (including using {{?RFC7919}}), TLS 1.2 implementations SHOULD
+               NOT negotiate cipher suites based on ephemeral finite-field
+               Diffie-Hellman key agreement (i.e., "TLS_DHE_*" suites).
```

Comment:
Should we clarify what we don't like about 7919: is it still vulnerable? Is it just not well deployed?

Reply:
explicitly state the reasons why it is not widely implemented / not usable in 1.2

Reply:
See 5f5c461